### ISSUE TITLE
The bilibili shortcode embeds a responsive video player for bilibili videos.But it is no longer applicable

### ISSUE THREAD

**Issue by OldHuang-C** [dillonzq/LoveIt#208]
The bilibili shortcode embeds a responsive video player for bilibili videos.But it is no longer applicable.
Because bilibili changed av to bv.Hope the author solves this problem.

**Comment by Jay-Young** [comment]
目前av和bv可以互转其实是没有问题，后续的bv视频如果不能转回av确实有这个问题。
修改themes/LoveIt/layouts/shortcodes/bilibili.html
```html
<div class="bilibili">
    {{- if .IsNamedParams -}}
        <iframe src="//player.bilibili.com/player.html?bvid={{ .Get `bv` }}&page={{ .Get `p` | default 1 }}" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"></iframe>
    {{- else -}}
        <iframe src="//player.bilibili.com/player.html?bvid={{ .Get 0 }}&page={{ .Get 1 | default 1 }}" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"></iframe>
    {{- end -}}
</div>
```

使用时改成如下两种写法:
```markdown
{{< bilibili 47027633 >}}
{{< bilibili bv=47027633 >}}
```

**Comment by dillonzq** [comment]
Fixed in #232
